### ISSUE TITLE
Please add support for Swift Package Manager.

### ISSUE THREAD

**Issue by OkiRules** [SwiftKickMobile/SwiftMessages#330]
Please add support for installing this project with the Swift Package Manager.

Thanks!

**Comment by danielsaidi** [comment]
Let me know if you want this done, but don't have the time to do it yourselves. I'll be more than happy to help, but don't know if it'd conflict with what you're planning in your other branches.

**Comment by wtmoose** [comment]
@danielsaidi I've love to get a PR on this one.

**Comment by danielsaidi** [comment]
@wtmoose I'll see what I can do :)

**Comment by danielsaidi** [comment]
@wtmoose I started converting the project to support spm and its folder structure (with a Sources and a Tests folder), but then noticed that SwiftMessages relies on xibs and built-in resources, which spm doesn't support. People could add these xibs and resources to their own projects, but I don't think that's the way it should be solved. I replaced xibs with code in my own projects and have no built-in resources...but I'm not sure if you want to go that way.
